doc/api/tls.md

```diff
@@ -911,10 +913,14 @@ added: v0.11.13
     the same order as their private keys in `key`. If the intermediate
     certificates are not provided, the peer will not be able to validate the
     certificate, and the handshake will fail.
-  * `ca`{string|string[]|Buffer|Buffer[]} Optional CA certificates to trust.
-    Default is the well-known CAs from Mozilla. When connecting to peers that
-    use certificates issued privately, or self-signed, the private root CA or
-    self-signed certificate must be provided to verify the peer.
+  * `ca` {string|string[]|Buffer|Buffer[]} Optionally replace the CA
+    certificates to trust. Default is the well-known CAs from Mozilla. Multiple
+    CAs can be provided using an array, and any string or Buffer can contain
+    multiple PEM certificates concatenated together. The peer's certificate
+    chain must be signed by a trusted CA for the connection to be authenticated.
+    When using certificates that are not signed by a well-known CA, the
+    certificate's root CA or the self-signed certificate must be specified as a
+    trusted CA or the connection will fail to authenticate.
```

Comment:
I attempted to clarify, by using less terminology, trying to use the terminology more consistently, and breaking out the self-signed clause as its own special case/sententce.

PTAL to see if your questions above are answered. I don't want to answer them inline in github, because if the documentation doesn't answer the questions, its not suceeding as documentation.